src/main/java/frc/trigon/robot/subsystems/intake/Intake.java

```diff
@@ -33,6 +33,7 @@ public void setBrake(boolean brake) {
     @Override
     public void stop() {
         masterMotor.stopMotor();
+        OperatorConstants.DRIVER_CONTROLLER.rumble(IntakeConstants.RUMBLE_DURATION_SECONDS, IntakeConstants.RUMBLE_POWER);
```

Comment:
This will also make it rumble if you let go every time it stops, even if you don't have a note. Find out how Yishai did it in Triumph's code